pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -238,8 +238,13 @@ ORDER BY JobName ASC
 
 // GetLastJobRunEndTimeFromTable retrieves the last imported job end time.
 func (c *ciDataClient) GetLastJobRunEndTimeFromTable(ctx context.Context, table string) (*time.Time, error) {
+	// Caution here, these tables can be large, especially for alerts. Do not query additional columns.
+	// We use the JobRunStartTime filter as this is the partition column to cut down on the size of the query.
+	// If we go more than 14 days without uploading data, we will have problems.
+	// At time of writing, this is a 120MB query for alerts, and 32MB for disruption.
```

Comment:
The query they replace was 13mb.

These are run once each per every 4 hours, an inconsequential amount of data for bigquery.